packages/ui/src/theme/css/component/searchField.scss

```diff
@@ -0,0 +1,26 @@
+.amplify-searchfield {
+  --amplify-components-fieldcontrol-color: var(
+    --amplify-components-searchfield-color
+  );
+  &__search {
+    color: var(--amplify-components-searchfield-search-color);
```

Comment:
it seems like this would create a circular dependency but this works out because the values are resolved higher up in the page (Thanks danny for this tip).

Reply:
![](https://media.giphy.com/media/vLruErVSYGx8s/giphy.gif)